pom.xml

```diff
@@ -108,12 +108,12 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
-      <version>2.1.17</version>
+      <version>2.2.0</version>
```

Comment:
Probably time for the [BOM](https://github.com/jenkinsci/bom#usage).

Reply:
https://github.com/jenkinsci/ssh-agent-plugin/pull/37